packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -215,7 +215,7 @@ class UpdatePackagesCommand extends FlutterCommand {
       // pub tool will attempt to bring these dependencies up to the most recent
       // possible versions while honoring all their constraints.
       final PubDependencyTree tree = new PubDependencyTree(); // object to collect results
-      final Directory temporaryDirectory = fs.systemTempDirectory.createTempSync('flutter_update_packages_');
+      final Directory temporaryDirectory = fs.systemTempDirectory.createTempSync('flutter_update_packages.');
```

Comment:
tempDir

Reply:
I feel dirty making this change, but, done.